src/Raven.Embedded/Raven.Embedded.csproj

```diff
@@ -15,10 +15,10 @@
   </PropertyGroup>
 
   <PropertyGroup Condition="'$(OS)' == 'Windows_NT'">
-    <TargetFrameworks>netstandard2.0;net462</TargetFrameworks>
+    <TargetFrameworks>netstandard2.0;net462;netcoreapp3.1</TargetFrameworks>
```

Comment:
we removed netcoreapp3.1 because it is deprecated

Reply:
Ok, would u accept net 5 or do u want it be net 6?

Reply:
is there any need for this not to be netstandard2.0 and net462 only? Doesn't it cover everything anyway?

Reply:
netstandard 2.0 requires the "System.Runtime.Loader" package where as net 3.1 & higher doesn't as it is provided natively the framework.

Reply:
have switched to net6.0